### ISSUE TITLE
Provision attempts to write keys before creating them

### ISSUE THREAD

**Issue by johnnyman727** [tessel/t2-cli#279]
```
~/self ❯❯❯ tessel provision                                                                                                        ⏎
INFO Connecting to Tessel...
INFO Connected to pixel over USB
INFO Creating public and private keys for Tessel authentication...
INFO Authenticating Tessel with public key...
INFO SSH Keys written.
```

Also, the keys were already written. This command should not have written over them.

**Comment by johnnyman727** [comment]
As seen by @MrNice.

**Comment by johnnyman727** [comment]
Here is how it should have looked (and it did after deleting `~/.tessel`)

```
~/self ❯❯❯ t2 provision
INFO Connecting to Tessel...
INFO Connected to pixel over USB
INFO Creating public and private keys for Tessel authentication...
INFO SSH Keys written.
INFO Authenticating Tessel with public key...
INFO Tessel authenticated with public key.
Add Comment Collapse
```

**Comment by wprater** [comment]
fixed by #295
